examples/app-pg/main.py

```diff
@@ -26,8 +25,10 @@
 
 
 def get_data(count: int) -> List[Tuple]:
+    from typing import List, Tuple
```

Comment:
Move imports to the top. 😞

Reply:
@kabirbaidhya dai mentioned to move it inside of the get_data function.

Reply:
:no_entry: **NO**